script/run-coverage.sh

```diff
@@ -10,8 +10,8 @@ fi
 
 # No test specified; run coverage on all of them
 if [[ -z "$1" ]] || [[ $1 == -* ]]; then
-  NODE_ENV=test $NYC --all --reporter=lcov --reporter=text --reporter=html mocha --reporter dot --reporter mocha-junit-reporter --no-color --opts src/platform/testing/unit/mocha.opts --recursive '{src,test}/**/*.unit.spec.js?(x)' src/platform/testing/unit/helper.js "$@"
+  BABEL_ENV=test $NYC --all --reporter=lcov --reporter=text mocha --reporter mocha-junit-reporter --no-color --opts src/platform/testing/unit/mocha.opts --recursive '{src,test}/**/*.unit.spec.js?(x)' src/platform/testing/unit/helper.js "$@"
   exit $?
 fi
 
-NODE_ENV=test $NYC --reporter=lcov --reporter=text --reporter=html mocha --reporter dot --reporter mocha-junit-reporter --no-color --opts src/platform/testing/unit/mocha.opts "$@"
+BABEL_ENV=test $NYC --reporter=lcov --reporter=text mocha --reporter mocha-junit-reporter --no-color --opts src/platform/testing/unit/mocha.opts src/platform/testing/unit/helper.js --recursive "$@"
```

Comment:
Did the helper and `--recursive` need to be added here?

Reply:
I'm not sure they did, but I did it to be consistent with how `mocha` is invoked in `run-unit-test.sh`.